tests/cases/unittests/services/preProcessFile.ts

```diff
@@ -183,7 +183,7 @@ describe('PreProcessFile:', function () {
                function foo() {
                }
                `, 
-                /* readImports */ false,
+                /* readImports */ true,

```

Comment:
because we previously did not check the list of ambient modules (fixed in this PR)